r/src/embedding.r

```diff
@@ -44,8 +44,30 @@ runEmbedding <- function(req) {
         df_embedding <- Embeddings(data, reduction = type)
     }
     # Order embedding by cells id in ascending form
-    df_embedding <- df_embedding[rownames(data@meta.data[order(data@meta.data$cells_id), ]), ]
```

Comment:
I assume this needs to be uncommented here.

Reply:
No, it's ok that it's deleted.

Reply:
Nope it should be good, we remake it and store it in `tmp`

Reply:
Tomorrow I'll add the code you sent me Alex

Reply:
I've actually uncommented this. The cell list order could be different than cellIds (I can't imagine how, but it's a possibility) so sorting the df should fix that.